extension/test/e2e/affirm.spec.js

```diff
@@ -65,118 +68,94 @@ describe.skip('::affirmPayment::', () => {
     async () => {
       const baseUrl = config.getModuleConfig().apiExtensionBaseUrl
       const clientKey = config.getAdyenConfig(adyenMerchantAccount).clientKey
-      let paymentAfterHandleRedirect
+      let paymentAfterCreateSession
+      let redirectPaymentResult
       try {
         const browserTab = await browser.newPage()
-        const paymentAfterMakePayment = await makePayment({
-          browserTab,
+        // Step #1 - Create a payment session
+        // https://docs.adyen.com/online-payments/web-components#create-payment-session
+        const createSessionRequest = await getCreateSessionRequest(
           baseUrl,
           clientKey,
-        })
+          'USD'
+        )
+
+        paymentAfterCreateSession = await createPaymentSession(
+          ctpClient,
+          adyenMerchantAccount,
+          ctpProjectKey,
+          createSessionRequest,
+          'USD'
+        )
         logger.debug(
-          'affirm::paymentAfterMakePayment:',
-          JSON.stringify(paymentAfterMakePayment)
+          'credit-card::paymentAfterCreateSession:',
```

Comment:
should it be affirm::paymentAfterCreateSession?

Reply:
Good catch
https://github.com/commercetools/commercetools-adyen-integration/pull/1064/commits/946c4d14c46c956e4250035afa1ff1c51d36c544